### ISSUE TITLE
Fuel percentage wrong

### ISSUE THREAD

**Issue by commputethis** [itchannel/fordpass-ha#518]
I wasn't sure how to title this.  Is this integration calculating the fuel remaining percentage or is it just pulling it from Ford?  I am assuming it is pulling it from Ford, but wanted to ask to see.  Mine is showing 103%.  I filled up and have driven about 10 miles.  If it is pulling from Ford, then there probably isn't much that can be done.  If it is being calculated by the integration, then there seems to be an issue and maybe assumptions made with the fuel tank size.  Anyway, I figured it was worth asking the question.  

![image](https://github.com/itchannel/fordpass-ha/assets/17625274/5d78ff85-4a93-4ea3-84fa-ed20e73068d0)

**Comment by TheLizard** [comment]
The value is directly passed from the fordpass API, I don't believe there is any calculation done by the integration:
```
            if self.sensor == "fuel":
                fuel_level = self.data.get("fuelLevel", {}).get("value")
                if fuel_level is not None:
                    return round(fuel_level)
                battery_soc = self.data.get("xevBatteryStateOfCharge", {}).get("value")
                if battery_soc is not None:
                    return round(battery_soc)
                return None
```

**Comment by commputethis** [comment]
That's what I assumed, but didn't want to assume. Thanks

**Comment by mclever** [comment]
A few people have noticed the same thing, I opened https://github.com/itchannel/fordpass-ha/issues/468. I think >100% should just show 100%.

**Comment by SquidBytes** [comment]
Yeah, its "known" because we are just returning the data and not calculating anything.
You could make a helper or manipulate it how you want, there are a few suggestions in #468. 
Personally I think the best option is to just return the data as it is, and let users do any calculations if they want which is why its remained that way, but it wouldn't be hard to cap it at 100%.

**Comment by commputethis** [comment]
I agree with what has been said and should be left the way it is.  Thanks for the responses.
